packages/terra-form/src/Control.scss

```diff
@@ -1,22 +1,26 @@
 @import '~terra-mixins';
```

Comment:
Good catch! This isn't needed at all for the project, so I removed it all together. https://github.com/cerner/terra-core/pull/646/commits/cc1792fce39e86d5056e2c948ae93b3d26db371a